server/api/src/utils.py

```diff
@@ -0,0 +1,21 @@
+import json
+
+
+def parse_yagna_key(response_json: str) -> (str, str):
```

Comment:
Unit tests for this function are highly welcome.

Reply:
But probably it's not needed at all – see comment on `server/yagna/init.sh`

Reply:
Tests added in https://github.com/golemfactory/yagna-service-poc/commit/e88222eb8a8c9bd7b44a88ca1eccf277c20ecc19